apps/meteor/app/slackbridge/server/SlackAdapter.js

```diff
@@ -1123,13 +1127,13 @@ export default class SlackAdapter {
 	@param [Date] timeStamp the timestamp the file was uploaded
 	**/
 	// details, slackMessage.file.url_private_download, rocketUser, rocketChannel, new Date(parseInt(slackMessage.ts.split('.')[0]) * 1000), isImporting);
-	uploadFileFromSlack(details, slackFileURL, rocketUser, rocketChannel, timeStamp, isImporting) {
+	async uploadFileFromSlack(details, slackFileURL, rocketUser, rocketChannel, timeStamp, isImporting) {
 		const requestModule = /https/i.test(slackFileURL) ? https : http;
 		const parsedUrl = url.parse(slackFileURL, true);
 		parsedUrl.headers = { Authorization: `Bearer ${this.apiToken}` };
-		requestModule.get(
+		await requestModule.get(
 			parsedUrl,
-			Meteor.bindEnvironment((stream) => {
+			await Meteor.bindEnvironment(async (stream) => {
```

Comment:
not sure this has the expected behavior..

`requestModule.get` 's second param is a callback.. looks like you're adding `await` because of you removed the `Promise.await` from `sendMessage`.. but the `await` you added will not `await sendMessage`, it will just `await Meteor.bindEnvironment`.. is that the case?

I guess you don't need this `await` and can do `void sendMessage`, since it is being called on `fileStore.insert`'s callback

Reply:
That's correct, thanks for the heads up, will remove that.